sdk/tests/conformance2/textures/misc/tex-storage-compressed-formats.html

```diff
@@ -29,17 +29,48 @@
 var wtu = WebGLTestUtils;
 var canvas = document.getElementById("canvas");
 var gl = wtu.create3DContext(canvas, null, 2);
+
+// Either of these extensions should be available on all WebGL 2.0 contexts
 var WEBGL_compressed_texture_etc = gl.getExtension("WEBGL_compressed_texture_etc");
-var vao = null;
+var WEBGL_compressed_texture_s3tc = gl.getExtension("WEBGL_compressed_texture_s3tc");
```

Comment:
const

Reply:
Done. Also changed dimensions to 4x4 because Firefox doesn't like 1x1 for compressed level 0.

Reply:
Oh! We should test that though. Let me check the specs.

Reply:
I think that's a bug in Firefox. The spec doesn't say anything explicitly, so I think it's implicitly allowed. (We have to support e.g. 1x1 mip:2 levels anyways)

Reply:
Bug filed: https://bugzilla.mozilla.org/show_bug.cgi?id=1615838